doc/guides/diagnostic-tooling-support-tiers.md

```diff
@@ -106,9 +106,13 @@ The tools are currently assigned to Tiers as follows:
 
 ## Tier 3
 
- | Tool Type | Tool/API Name             | Regular Testing in Node.js CI | Integrated with Node.js | Target Tier |
- |-----------|---------------------------|-------------------------------|-------------------------|-------------|
- |           |                           |                               |                         |             |
+ | Tool Type | Tool/API Name                        | Regular Testing in Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|--------------------------------------|-------------------------------|-------------------------|-------------|
+ | Profiling | V8 CPU profiler                      | Partial (V8 Tests)            | Yes                     |     1       |
+ | Profiling | --prof/--prof-process flags          | Yes                           | Yes                     |     1       |
```

Comment:
How come these flags are separate to the V8 CPU profiler? And the current tests do not properly test everything in depth, right?

Reply:
> How come these flags are separate to the V8 CPU profiler?

Because they are separate things: V8 CpuProfiler is a C++ API officially supported by V8 which can also be used through the inspector protocol. --prof is a flag not officially supported by V8. 

> And the current tests do not properly test everything in depth, right?

Right. That's why we're moving them to Tier 3 and not 2/1.

Reply:
Should we maybe say it's also partial in that case until we have more tests in place?